### ISSUE TITLE
sst:end fires right after sst:start

### ISSUE THREAD

**Issue by timsim00** [YerkoPalma/choo-stt#3]
Can't seem to get this to work (chrome/OSX).  However, [this demo](https://www.google.com/intl/en/chrome/demos/speech.html) is working.  Also, browser is not asking permission to use microphone when running example for choo-stt.

**Comment by YerkoPalma** [comment]
That's weird. Are you running the example from this repo? if not, could you post the code you're using?

**Comment by timsim00** [comment]
Yes, just cloned, npm install, npm start. Node 6.9.4 and 8.6.0.  state.stt.serviceURI is undfined.  In the console I can "new webkitSpeechRecognition()".

**Comment by timsim00** [comment]
Got it working. For my version of chrome (62.0.3202.94) there doesn't seem to be a way to blanket allow all sites access to the [microphone](chrome://settings/content/microphone).  So I have to ask permission with [getUserMedia()](https://www.html5rocks.com/en/tutorials/getusermedia/intro/).  But this now has to be done only via ssl, so I had to run choo-stt using bankai or figure out budo ssl.  So I ran with bankai, requested microphone from console, clicked start, and it worked.

**Comment by YerkoPalma** [comment]
In my version of chrome, it does ask for permission for the mic, withouth having to call `getUserMedia()` or set ssl. It is good to know, I'll keep this open for now, perhaps is good to mention in docs
